services/motion/builtin/builtin_test.go

```diff
@@ -555,9 +556,7 @@ func TestMoveOnGlobe(t *testing.T) {
 			extra,
 		)
 		test.That(t, err, test.ShouldBeNil)
-		plan, err := moveRequest.plan(ctx)
-		test.That(t, err, test.ShouldBeNil)
-		waypoints, err := plan.GetFrameSteps(fakeBase.Name().Name)
```

Comment:
why is this no longer needed?

Reply:
The return type of plan changed

Reply:
Perhaps we should have `SingleComponentPlan` and `MultiComponentPlan`

Reply:
Since we just introduced these new types, I want to see how we use them and organically adapt them instead of trying to foresee both use cases.  Presently we have no need to return `motionplan.Plan` so I switched to this type, but in the future we might want to move this way

Reply:
Actually I could be convinced that it would be more correct to try to do this now.  I'm on the fence because it seems like more work to do in the short term but we'd probably rather have the full plan here in the future.